packages/legend-graph/src/graphManager/protocol/pure/v1/transformation/pureGraph/to/helpers/V1_FlatDataStoreBuilderHelper.ts

```diff
@@ -169,9 +169,11 @@ const buildFlatDataProperty = (
     `Flat-data property 'value' field is missing`,
   );
   assertTrue(
-    isString(property.value) ||
-      isNumber(property.value) ||
-      isBoolean(property.value),
+    property.value.reduce(
```

Comment:
I think we can use `Array.every` here instead

Reply:
fixed